backend/src/main/java/co/inventorsoft/scripty/controller/MockRequestController.java

```diff
@@ -0,0 +1,79 @@
+package co.inventorsoft.scripty.controller;
+
+import co.inventorsoft.scripty.model.dto.MockRequestDto;
+import co.inventorsoft.scripty.model.dto.StringResponse;
+import co.inventorsoft.scripty.service.MockRequestService;
+import com.fasterxml.jackson.core.JsonParseException;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.InvalidMimeTypeException;
+import org.springframework.web.bind.annotation.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import java.nio.charset.Charset;
+import java.nio.charset.IllegalCharsetNameException;
+import java.util.regex.Pattern;
+
+/**
+ * @author A1lexen
+ */
+
+@Api(description = "Operations with mock requests")
+@RestController
+public class MockRequestController {
+    @Value("${prefix.link}")
```

Comment:
`@Value` is part of dependency injection, so please move this assignment to constructor